training/templates/training/join.html

```diff
@@ -2,19 +2,23 @@
 
 {% block content %}
 
-<div class="donemessage">
-    Congratulations: you are successfully registered in <b>{{ training.training_identifier }}</b><br/>
-    <a href="https://{{ host }}">Return to Galaxy</a>
-</div>
+<div class="container">
+
+	<p class="alert alert-success text-center">
+		Congratulations: you are successfully registered in <b>{{ training.training_identifier }}</b><br/>
+	</p>
+
+	<br>
 
-<h2>How It Works</h2>
-<p>
-We have deployed a dedicated compute node just for your training session to
-use. No one outside of your training has access to this machine. Completely
-transparently to you, all of the jobs that you run in Galaxy, during the
-period of the training event, will "prefer" to run on this machine. If there is
-no room on that machine, they will run on any other machine in our cluster with
-resources.
-</p>
+	<p class="lead text-center">
```

Comment:
I'd like to keep something of this here, because this page is mostly viewed by students, it'll be the first time they hear about TIaaS, otherwise they're just visiting some random URL and don't know why if the teacher doesn't explain it.

So having a short blurb here introduces them to the concept, why they've had to click this link (if the teacher didn't explain which is common), and lets them understand what's going on.

Reply:
Oh agreed, I haven't removed this it's just been moved to the "How does it work" snippet:

```html
<!-- join.html -->
<div class="container">
	<p class="alert alert-success text-center">
		Congratulations: you are successfully registered in <b>{{ training.training_identifier }}</b><br/>
	</p>

	<br>

	<p class="lead text-center">
		<a href="{{ TIAAS_DOMAIN }}">Return to Galaxy</a>
	</p>

	<br><br>

	<div class="container" style="max-width: 800px;">
		{% include 'snippets/about/4_how_does_it_work.html' %}
	</div>
</div>
```

Reply:
Oh! I missed that, apologies